client/js/widgets/widget.js

```diff
@@ -1463,7 +1463,10 @@ export class Widget extends StateManaged {
       if(typeof w1.get(key) == 'number')
         return w1.get(key) - w2.get(key);
       else
-        return w1.get(key).localeCompare(w2.get(key), locales, options);
+        if(w1.get(key) !== null && w2.get(key) !== null)
+          return w1.get(key).localeCompare(w2.get(key), locales, options);
+        else
+          return (w1.get(key) === null && w2.get(key) === null) ? 0 : (w1.get(key) === null ? -1 : 1)
```

Comment:
Perhaps clearer is
```
if(w1.get(key) !== null 
          return w2.get(key) !== null ? w1.get(key).localeCompare(w2.get(key), locales, options) : 1;
else
          return w2.get(key) !== null) ? -1 : 0
```

Reply:
Yeah, I think that is better, but would it be more clear if I got rid of all the !== like this?  I was at first thinking of using the !== as a way of preventing null values from getting to the localCompare (and causing exceptions), but I think this would do that as well.

if(w1.get(key) === null )
          return w2.get(key) === null) ?  0 : -1;
else
          return w2.get(key) === null  ? 1 : w1.get(key).localeCompare(w2.get(key), locales, options) ;

Reply:
Looks fine.

Reply:
(If you make one of those changes, that is).